lib/std/fmt.zig

```diff
@@ -1026,102 +1032,6 @@ pub fn formatBuf(
     }
 }
 
-/// Print a float in scientific notation to the specified precision. Null uses full precision.
-/// For floats with less than 64 bits, it should be the case that every full precision, printed
-/// value can be re-parsed back to the same type unambiguously.
-///
-/// Floats with more than 64 are currently rounded, see https://github.com/ziglang/zig/issues/1181
-pub fn formatFloatScientific(
```

Comment:
These could use a compileError to deprecate. Alternatively, we could provide this as a slim wrapper over the actual ryu implementations (which are not currently exposed).